src/Core/tests/Scope/UseCaseTest.php

```diff
@@ -79,7 +79,7 @@ public function testScopeBindingsAsNotSingletons(bool $theSame, string $alias, m
 
     public static function provideScopeBindingsAsNotSingletons(): iterable
     {
-        yield 'array-factory' => [false, 'foo', [Factory::class, 'makeStdClass']];
+        yield 'array-factory' => [false, 'foo', (new Factory())->makeStdClass(...)];
```

Comment:
Need to revert it because we test callable arrays in this test case

Reply:
reverted https://github.com/spiral/framework/pull/1177/commits/38ee09a40f18f496be2d2e5da6d00ecbbfe414bf 👍